pkgs/applications/virtualization/xhyve/default.nix

```diff
@@ -1,15 +1,15 @@
-{ stdenv, lib, fetchurl, Hypervisor, vmnet, xpc, libobjc }:
+{ stdenv, lib, fetchurl, Hypervisor, vmnet, xpc, libobjc, zlib }:
 
 stdenv.mkDerivation rec {
   name    = "xhyve-${version}";
-  version = "1f1dbe305";
+  version = "1dd9a5165";
```

Comment:
For packages without versions we use the commit date, otherwise `nix-env -u` doesn't work properly.

Reply:
@jsamsa Mind fixing this? :-)

Reply:
Sure thing. Just amended my commit after running nix-build.